build/lib/SpinCore_pp/SpinCore_pp.py

```diff
@@ -209,13 +212,21 @@ def runBoard():
     return _SpinCore_pp.runBoard()
 runBoard = _SpinCore_pp.runBoard
 
-def getData(output_array, nPoints, nEchoes, nPhaseSteps, output_name):
-    return _SpinCore_pp.getData(output_array, nPoints, nEchoes, nPhaseSteps, output_name)
+def getData(output_array, nPoints, nEchoes, nPhaseSteps):
+    return _SpinCore_pp.getData(output_array, nPoints, nEchoes, nPhaseSteps)
```

Comment:
don't know how this got missed - thought we did this a while ago but we should check that this still works (check pyspecdata getData and make sure this won't error out)